foundations/javascript_basics/project_calculator.md

```diff
@@ -46,3 +48,5 @@ Here are some use cases (abilities your project needs to have):
 * Add a "backspace" button, so the user can undo if they click the wrong number.
 * Add keyboard support!
 </div>
+
+### POST-ASSIGNMENT SECTION HEADING
```

Comment:
```suggestion

```
I don't believe this is necessary. It also would just say `POST-ASSIGNMENT SECTION HEADING` which wouldn't look very good. You're supposed to write something else, but I don't think anything would go here.